docs/azure/GettingStarted/Deploy.rst

```diff
@@ -87,21 +70,36 @@ To install the nRF Asset Tracker into your Azure account, complete the following
 
       az provider register --namespace Microsoft.AzureActiveDirectory
 
-#. Create an Azure Active Directory B2C. Currently, it is not possible to create an Active Directory B2C and application through the ARM template (see `GitHub issue <https://github.com/NordicSemiconductor/asset-tracker-cloud-azure-js/issues/1>`_).
+#. Follow the `tutorial for creating an Azure Active Directory B2C <https://docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-create-tenant>`_. On the creation screen, select the resource group you have created above. Currently, it is not possible to create an Active Directory B2C and application through the ARM template (see `GitHub issue <https://github.com/NordicSemiconductor/asset-tracker-cloud-azure-js/issues/1>`_).
+
+#. Save the initial domain name of the created Active Directory B2C to the environment variable ``B2C_TENANT``.
+   In this example, ``nrfassettrackerusers`` is used as the initial domain name.
+
+   .. code-block:: bash
 
-   a. Follow the instructions in the `tutorial for registering a web application in Azure Active Directory B2C <https://docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-register-applications?tabs=applications>`_ and create a B2C tenant and an application. Use ``http://localhost:3000/`` (for local development) and ``https://<your APP_NAME>app.z16.web.core.windows.net/`` as the redirect URLs.
+      # add to .envrc
+      export B2C_TENANT="nrfassettrackerusers"
 
-   #. Enable the implicit grant and hybrid flows for :guilabel:`Access tokens` and :guilabel:`ID tokens` and click :guilabel:`Save`.
+#. After creating the Active Directory B2C, under :guilabel:`User flows`, create the user flow for sign up and sign in (recommended version), and make sure to name the userflow as ``B2C_1_signup_signin``.
 
-#. Create the user flow for sign-up, sign-in, and make sure to name the userflow as ``B2C_1_signup_signin``.
+#. Follow the instructions in the `tutorial for registering a web application in Azure Active Directory B2C <https://docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-register-applications?tabs=app-reg-ga#register-a-web-application>`_ and register a web application. Use ``https://<your APP_NAME>app.z16.web.core.windows.net/`` as the redirect URL.
 
-   a. Click :guilabel:`Application claims`, select :guilabel:`Show more ...` and then mark :guilabel:`Email Addresses` as a return claim.
+#. After registering the application, under :guilabel:`Authentication`, enable the implicit grant and hybrid flows for :guilabel:`Access tokens` and :guilabel:`ID tokens` and click :guilabel:`Save`.
```

Comment:
```suggestion
#. Select :guilabel:`Authentication`, enable the implicit grant and hybrid flows for :guilabel:`Access tokens` and :guilabel:`ID tokens` and click :guilabel:`Save`.
```
Where is the Authentication exactly?

Reply:
![image](https://user-images.githubusercontent.com/188915/119957681-95935500-bfa2-11eb-9400-8a136cd5d674.png)